.husky/pre-push

```diff
@@ -1,4 +0,0 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
-
-npm test
```

Comment:
I'm not sure how to undo the deletion of the pre-push file. Could you help me with that @SamuelAlev ?

Reply:
`git checkout main .husky/pre-push` and then commit it back